tests/scripts/webhook-config.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: ${WEBHOOK_CONFIG_NAME}
   namespace: ${NAMESPACE}
 webhooks:
-  - name: ${SERVICE_NAME}.${NAMESPACE}.svc
+  - name: cephcluster-webhook.${SERVICE_NAME}.${NAMESPACE}
```

Comment:
Why you removed the ".svc" suffix?

Reply:
In my understanding, the web hook names are not referenced at all, and the naming convention does not exist. 
And I thought using `.svc` suffix was redundant because it does not provide any information.
By googling, people are using something like domain names. ( for example, `kb.io`, `.xxx.com`)

So it may be better to use `rook.io` as the suffix of the web hook names.